### PR TITLE
Feature/dwpad

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: experimental/xformer/Transforms/ConvPatternsTFL.cpp [hunk 1]
```diff
@@ -184,20 +95,32 @@ LogicalResult ReplaceConv2DBase<ConcreteType, TFLConvOpType>::getArgs(
...
-          conv2DOp.stride_w(), opPadding, &newWidth, &padLeft,
-          &padRight) != tensorflow::Status::OK()) {
-    return failure();
+
+  if (conv2DOp.padding() == "EXPLICIT") {
+    auto paddingValuesConstOp =
+        dyn_cast<mlir::ConstantOp>(conv2DOp.padding_values().getDefiningOp());
+    auto paddingValues =
+        paddingValuesConstOp.value().template cast<DenseElementsAttr>();
+    padTop = paddingValues.template getValue<int32_t>({1, 0});
```
Comment (andrewstanfordjason) [inline experimental/xformer/Transforms/ConvPatternsTFL.cpp:109]: where did these numbers come from?

**Review by andrewstanfordjason — COMMENTED** [review]
looks good to me except for the EXPLICIT stuff, I don't think I understand

File: experimental/xformer/Transforms/ConvPatternsTFL.cpp (hunk 1 above)
Comment (ghost) [reply experimental/xformer/Transforms/ConvPatternsTFL.cpp:109]: I'll add comments for that. The padding values for the PadOp are stored as a 4x2 tensor, which 1,0 and 1,1 being top and bottom, and 2,0 and 2,1 being left and right. 0,0 and 0,1 would be for the batch dimension and 3,0, and 3,1 for the channel/depth.

Comment (ghost) [reply experimental/xformer/Transforms/ConvPatternsTFL.cpp:109]: Does that make sense?

Comment (andrewstanfordjason) [reply experimental/xformer/Transforms/ConvPatternsTFL.cpp:109]: yes, that makes sense. A comment would be helpful though

**Review by andrewstanfordjason — APPROVED** [review]

**Pull request merged**